src/media.rs

```diff
@@ -50,6 +50,8 @@ impl FromStr for Media {
 }
 
 const TABLE: &[(&str, Media, &[&str])] = &[
+  ("application/json", Media::Text, &["json"]),
+  ("application/yaml", Media::Text, &["yaml"]),
```

Comment:
`.yml` is a sometimes used file extension too

Reply:
Seems reasonable to me! Added.